x/observer/simulation/operation_disable_cctx.go

```diff
@@ -0,0 +1,55 @@
+package simulation
+
+import (
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/zeta-chain/node/x/observer/keeper"
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+// SimulateDisableCCTX generates a MsgDisableCCTX and delivers it.
+func SimulateDisableCCTX(k keeper.Keeper) simtypes.Operation {
+	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accounts []simtypes.Account, _ string,
+	) (OperationMsg simtypes.OperationMsg, futureOps []simtypes.FutureOperation, err error) {
+		policyAccount, err := GetPolicyAccount(ctx, k.GetAuthorityKeeper(), accounts)
+		if err != nil {
+			return simtypes.NoOpMsg(types.ModuleName, types.TypeMsgDisableCCTX, err.Error()), nil, nil
+		}
+		authAccount := k.GetAuthKeeper().GetAccount(ctx, policyAccount.Address)
+		spendable := k.GetBankKeeper().SpendableCoins(ctx, authAccount.GetAddress())
+
+		msg := types.MsgDisableCCTX{
```

Comment:
Can't we had randomness to disable outbound as well

Reply:
We can , however I do not want to add more disable Inbound messages into the simulation as the Disable inbound is already a problem as mentioned above 
https://github.com/zeta-chain/node/pull/3207#discussion_r1884281128

If we keep DisableInbound:  false .  DisableOutbound has to be true .